cirq-core/cirq/ops/common_gates.py

```diff
@@ -906,18 +906,18 @@ def __repr__(self) -> str:
 
 
 class CZPowGate(gate_features.InterchangeableQubitsGate, eigen_gate.EigenGate):
-    """A gate that applies a phase to the |11⟩ state of two qubits.
+    r"""A gate that applies a phase to the |11⟩ state of two qubits.
 
     The unitary matrix of `CZPowGate(exponent=t)` is:
 
-        [[1, 0, 0, 0],
-         [0, 1, 0, 0],
-         [0, 0, 1, 0],
-         [0, 0, 0, g]]
-
-    where:
-
-        g = exp(i·π·t).
+    $$
+    \begin{bmatrix}
+        1 & 0 & 0 & 0 \\
```

Comment:
This nit applies throughout: A lot of the matrices have few non-zero elements and they are easier to read if the zeros are not explicitly shown:

```
$$
\begin{bmatrix}
    1 & & & \\
    & 1 & & \\
    & & 1 & \\
    & & & e^{i \pi t} \\
\end{bmatrix}
$$

Reply:
For this one I'm going to open an issue.  I struggled with this myself.  For three qubit gates the sparse representation is a definite yes.  But I worry a bit that not showing the 0s will confuse users